l10n_it_fatturapa_pec/tests/test_e_invoice_response.py

```diff
@@ -98,6 +98,7 @@ def test_process_response_INVIO(self):
         e_invoices = self.attach_in_model.search([]) - e_invoices
 
         self.assertTrue(e_invoices)
+        self.assertTrue(e_invoices.e_invoice_received_date)
```

Comment:
@robyf70 puoi verificarne il valore?

Reply:
@eLBati Per soddisfare il test possiamo controllare che abbiano la stessa data, controllare anche l'orario non avrebbe senso

Reply:
Allora forse `e_invoice_received_date` dovrebbe essere Date invece di Datetime

Reply:
@eLBati Penso sia utile avere l'informazione completa dalla PEC, troncare o meno l'orario dalla data di ricezione non cambia nulla, potremmo avere in entrambi i casi un problema solo quando i test vengono eseguiti a cavallo di mezzanotte

Reply:
> @robyf70 puoi verificarne il valore?

Ho trovato il modo di verificare la data completa di orario